### PR TITLE
Refs #31777 -- Doc'd Char/TextField.db_collation parameter in MySQL notes.

### PR DESCRIPTION
**Pull request by felixxm**
ticket-31777

I've noticed that we missed this section in e387f191f76777015b6ea687ce83cdb05ee47cee.

@knyghty Tom, Can you take a look?

### CODE REVIEW

File: docs/ref/databases.txt [hunk 1]
```diff
@@ -398,11 +398,15 @@ Collation settings
...
-level and also per-table and per-column. This is `documented thoroughly`_ in
-the MySQL documentation. In all cases, you set the collation by directly
-manipulating the database tables; Django doesn't provide a way to set this on
-the model definition.
+as well as what strings compare as equal. You can specify the ``db_collation``
+parameter to set the collation name of the column for
+:attr:`CharField <django.db.models.CharField.db_collation>` and
+:attr:`TextField <django.db.models.TextField.db_collation>`.
+
+It can also be set on a database-wide level and per-table. This is
```
Comment (carltongibson) [inline docs/ref/databases.txt:406]: I'm not sure `It` carries from the previous paragraph...

```suggestion
The collation can also be set on a database-wide level and per-table. This is
```

Wrapping?

**Review by carltongibson — APPROVED** [review]
Seems reasonable. Thanks. 👍

**Review by pope1ni — APPROVED** [review]
Good catch. LGTM.

File: docs/ref/databases.txt (hunk 1 above)
Comment (felixxm) [reply docs/ref/databases.txt:406]: Updated.

**Comment by felixxm** [comment]
Thanks both :rocket:

**Review by knyghty — APPROVED** [review]
👍  (reminds me I should think about if we want db_collation Model Meta option for vendors that support it)

**Pull request merged**